drivers/pwm/pwm_mcux_ftm.c

```diff
@@ -441,6 +449,9 @@ static const struct pwm_driver_api mcux_ftm_driver_api = {
 
 #define TO_FTM_PRESCALE_DIVIDE(val) _DO_CONCAT(kFTM_Prescale_Divide_, val)
 
+#define PINCTRL_DEFINE(n) PINCTRL_DT_INST_DEFINE(n);
+#define PINCTRL_INIT(n) .pincfg = PINCTRL_DT_INST_DEV_CONFIG_GET(n),
```

Comment:
why do we need this in this case? This driver seems to be using pinctrl unconditionally.

Reply:
We don't, I missed this. I've dropped the defined and used the macros directly, thanks for noticing.